pkg/log/options.go

```diff
@@ -185,14 +153,16 @@ func (o *Options) AttachCobraFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().BoolVar(&o.JSONEncoding, "log_as_json", o.JSONEncoding,
 		"Whether to format output as JSON or in plain console-friendly format")
 
-	cmd.PersistentFlags().StringVar(&o.outputLevel, "log_output_level", o.outputLevel,
-		"The minimum logging level of messages to output, can be one of debug, info, warning, error, or none")
+	cmd.PersistentFlags().StringVar(&o.outputLevel, "log_output_level", string(o.outputLevel),
+		fmt.Sprintf("The minimum logging level of messages to output, can be one of %s, %s, %s, %s, or %s})",
```

Comment:
nit: use `%q` here to print the strings with quotes? `can be one of "debug", "info", ...`

Reply:
done.